includes/admin/assets.php

```diff
@@ -69,7 +69,19 @@ public function load()
 			SIMPLE_CALENDAR_VERSION,
 			true
 		);
-
+		wp_register_script(
+			'simcal-sc-oauth-helper-admin',
+			$js_path . 'sc-oauth-helper-admin.min.js',
+			['jquery'],
+			SIMPLE_CALENDAR_VERSION,
+			true
+		);
+		$run_oauth_helper = get_option('simple_calendar_run_oauth_helper');
+		if($run_oauth_helper){
```

Comment:
We are running the JS irrespective of the page, shouldn't we load this script only when settings page is being viewed?

Reply:
Added condition